vcd/resource_vcd_vapp_vm.go

```diff
@@ -312,6 +313,40 @@ func resourceVcdVAppVmUpdate(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("error getting VM status: %#v", err)
 	}
 
+	if d.HasChange("metadata") {
+		oraw, nraw := d.GetChange("metadata")
```

Comment:
Please fix same naming variable issues as mentioned in previous comments.

Reply:
Fixed.